README.md

```diff
@@ -23,28 +32,26 @@ OpenShift Origin is a distribution of [Kubernetes](https://kubernetes.io) optimi
 **Learn More:**
 
 * **[Public Documentation](http://docs.openshift.org/latest/welcome/index.html)**
-* The **[Trello Roadmap](https://ci.openshift.redhat.com/roadmap_overview.html)** covers the epics and stories being worked on (click through to individual items)
+  * **[API Documentation](http://docs.openshift.org/latest/rest_api/openshift_v1.html)**
 * **[Technical Architecture Presentation](https://docs.google.com/presentation/d/1Isp5UeQZTo3gh6e59FMYmMs_V9QIQeBelmbyHIJ1H_g/pub?start=false&loop=false&delayms=3000)**
 * **[System Architecture](https://github.com/openshift/openshift-pep/blob/master/openshift-pep-013-openshift-3.md)** design document
-* **[API Documentation](http://docs.openshift.org/latest/rest_api/openshift_v1.html)**
+* The **[Trello Roadmap](https://ci.openshift.redhat.com/roadmap_overview.html)** shows work (epics and user stories) that are planned or currently underway (click through to individual items)
```

Comment:
Moved the Trello Roadmap down, because I think the docs are most relevant for users